route/grafananet.go

```diff
@@ -163,7 +171,7 @@ func NewGrafanaNet(key string, matcher matcher.Matcher, cfg GrafanaNetConfig) (R
 	if cfg.AggregationFile != "" {
 		aggregation, err = conf.ReadAggregations(cfg.AggregationFile)
 		if err != nil {
-			return nil, fmt.Errorf("NewGrafanaNet: could not read aggregationFile %q: %s", cfg.AggregationFile, err.Error())
+			return nil, err
```

Comment:
why? and why do we do it differently here than in NewGrafanaNetConfig() ?

Reply:
This just wraps the original  (pre-PR) `ReadAggregations(...)` call in an if-else because it's possible that no file will be supplied. 

I see you're commenting against a diff within two commits of the PR though:
![image](https://user-images.githubusercontent.com/15159201/155767429-44521989-c96c-4780-a3d7-8d70eaa599da.png)
In an earlier commit, I added the `"NewGrafanaNet:..."` error since I'd removed the aggregation file parameter from `NewGrafanaNetConfig()`. After the suggestion to keep the aggregation file parameter, I reverted that change and then reverted the error in this method back to what it was originally - just returning the error from reading the aggregations.